client/js/lounge.js

```diff
@@ -448,6 +452,12 @@ $(function() {
 				.find(".unread-marker")
 				.appendTo(container);
 		}
+		if (data.msg.type === "channel_list") {
+			// Delay by 1 second so the browser has time to render all channels
+			$(target + " .chat").stop().delay(1000).animate({
+				scrollTop: 0
+			});
+		}
```

Comment:
@astorije I'm running the server on my VPS and loading takes about ~10 seconds (on v2.2.2), so that might be your RPi.

The delay is needed client side because the browsers needs a while (about 150-400ms here) to render the table with 500 rows. The 1 second is arbitrary.

@xPaw The table is sorted descending meaning channels with the most users (and most interesting channels) can be found on top. Having to scroll through them manually isn't very user friendly.